### PR TITLE
Removed newer @extend usage

### PR DESCRIPTION
**Pull request by MichaelArestad**
This is a part 2 of the previous https://github.com/Automattic/dops-components/pull/26

This one actually removes the rest of the @extends that were brought in after the other PR was made.

### CODE REVIEW

File: client/scss/calypso-form.scss [hunk 1]
```diff
@@ -10,6 +10,77 @@
 }
 
 %form-field {
+	// margin: 0;
```
Comment (enejb) [inline client/scss/calypso-form.scss]: Can you just remove these lines.

Comment (MichaelArestad) [inline client/scss/calypso-form.scss]: Removing.

File: client/scss/extends.scss [hunk 2]
```diff
@@ -6,14 +6,16 @@
 }
 
 %noticon {
-	@extend %clear-text;
+	-webkit-font-smoothing: antialiased;
```
Comment (enejb) [inline client/scss/extends.scss]: Can we not just remove this file now?

**Comment by enejb** [comment]
:shipit:

**Pull request merged**